client/src/components/Dashboard.js

```diff
@@ -27,6 +28,7 @@ const Dashboard = () => {
             <ReportJackpot props={{roomNames, jackpots}}/>
             <h1>Biggest Current Jackpot</h1>
             <BiggestCurrentJackpot />
+            <NewsFeed />
```

Comment:
Try cleaning up the ternary on line 24 with variables in the component to something like this, where you set up variables for what you conditionally render.

```js
{roomNames.map((room, i) => {
  const isJackpotHit = jackpots[i].hit ? 'Yes' : 'No';
  const lastReport = `${room} Last Report - Hit: ${isJackpotHit} - Amount: $${jackpots[i].amount}`;
  const firstJackpotMessage = `${room} No reported Jackpots Yet`;
  return (
    <li key={room}>{jackpots[i] ? lastReport : firstJackpotMessage}</li>
  )
})}
```

Reply:
love it thanks.